skycoin/skywire
# Add Windows Installer Job to AppVeyor

Did you run `make format && make check`?

Fixes #1149 

 Changes:	
- use standard `wix` library instead `go-msi`
- add windows installer creator process to AppVeyor job
- separate goreleaser for each OSs (linux/darwin/windows)
- add desktop shortcut for windows installer and make its installer a few pretty by header and dialog image

How to test this PR:
- This is test release on my fork https://github.com/mrpalide/skywire/releases
- and this is AppVeyor logs for checking https://ci.appveyor.com/project/mrpalide/skywire/builds/43983928
- for test wix standard library, use `make win-installer CUSTOM_VERSION=v1.0.0-rc7` for now
- also for test `git push origin TAG` you should make a PR from my branch on yours and then test it
